index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+      </head>
+    <body>
+     <div class="container">
+      <header>
+        <nav class="nav">
+          <img src="images/logo.png" alt="my logo"/>
```

Comment:
- [ ] I would kindly recommend you wrap your logo in an `a` tag to make your site more engaging. so that users can click on it.